bmorph/evaluation/format_data.py

```diff
@@ -0,0 +1,60 @@
+KACimport numpy as np
```

Comment:
Because this script is study specific I think we can remove it, and just keep it in our personal, project-specific repos. Also, `KAC` at the beginning would throw an error at import.

Reply:
great catch, I think I was typing in the wrong window